.claude/skills/example-app-patterns/SKILL.md

```
@@ -0,0 +1,243 @@
+
+### Data Flow
+
+```
+accordionData[].subCategory[].action  -->  handleItemAction(item) switch
+                                              |
+                        +---------------------+---------------------+
+                        |                                           |
+                  Simple/inline                              Complex/reusable
+              CleverTap.method()                          AppUtils.helperName()
+              directly in switch                          (defined in app-utils.js)
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)